api/tests/Api/ContentNodes/RootColumnLayout/CreateRootColumnLayoutTest.php

```diff
@@ -35,7 +35,7 @@ public function testCreateColumnLayoutValidatesMissingParent() {
         $this->assertJsonContains([
             'violations' => [
                 [
-                    'propertyPath' => 'parent',
+                    'propertyPath' => 'root',
```

Comment:
The test name still says the parent may not be missing, not the root. Should this be a new test instead? Or do we just need to adjust the test's name?

Reply:
Unfortunately no. 
`parent` is the mandatory field. But without `parent` `root` is not determined. 
Unfortunately `root` is checked first. 
It is correct that a ValidationError occurs. However, the message is unfortunate. 
Unfortunately, I have not found a better solution.